packages/vitest/src/defaults.ts

```diff
@@ -1,10 +1,10 @@
 import type { BenchmarkUserOptions, ResolvedCoverageOptions, UserConfig } from './types'
 import { isCI } from './utils/env'
 
-export const defaultInclude = ['**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}']
+export const defaultInclude = ['**/*.{test,spec}.{,m,c}{j,t}sx?']
```

Comment:
Is `x?` valid on glob or is it just for regexp?

This works:
```suggestion
export const defaultInclude = ['**/*.{test,spec}.{,m,c}{j,t}s{,x}']
```

Reply:
Yea, I got the syntax wrong, though I think `?(x)` looks better